repositories.bzl

```diff
@@ -22,9 +22,9 @@ airy_jvm_deps = [
     "javax.activation:javax.activation-api:1.2.0",
     "javax.validation:validation-api:2.0.1.Final",
     "javax.xml.bind:jaxb-api:2.3.1",
-    "org.apache.logging.log4j:log4j-core:2.12.1",
-    "org.apache.logging.log4j:log4j-slf4j-impl:2.12.1",
-    "org.slf4j:slf4j-api:1.7.29",
+    "org.apache.logging.log4j:log4j-core:2.15.0",
```

Comment:
Good change. Now you only need to run
```
bazel run @unpinned_maven//:pin
```
to pin those changes in the `maven_install.json` file :)